driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -840,41 +830,26 @@ public Builder notStarted() {
         }
 
         /**
-         * Sets this cluster to be a DSE cluster (defaults to {@link #isDSE()} if this is never called).
-         */
-        public Builder withDSE() {
-            this.isDSE = true;
-            return this;
-        }
-
-        /**
-         * Sets this cluster to be a non-DSE cluster (defaults to {@link #isDSE()} if this is never called).
+         * The Cassandra version to use.
          */
-        public Builder withoutDSE() {
-            this.isDSE = false;
+        public Builder withCassandraVersion(VersionNumber versionNumber) {
```

Comment:
Again, not sure this makes sense, since `withCassandraVersion` and `withDSEVersion` are mutually exclusive (see assertions) why not keep just one method `withVersion`?